### PR TITLE
Added Thirus's Dynamite to the DailyTasks plugin.

### LINKED ISSUE #7462: Add Thirus' Dynamite to the daily tasks indicator
title

### PR DESCRIPTION
**Pull request by langhillie**
Added Thirus' daily dynamite from the Kourend diaries to the DailyTasks plugin.

Closes #7462

### CODE REVIEW

**Comment by langhillie** [comment]
Any update on this?

**Comment by Zaino60** [comment]
Any update on this?

File: runelite-client/src/main/java/net/runelite/client/plugins/dailytaskindicators/DailyTasksConfig.java [hunk 1]
```diff
@@ -109,4 +109,15 @@ default boolean showBonemeal()
 	{
 		return true;
 	}
+
+	@ConfigItem(
+			position = 8,
+			keyName = "showDynamite",
+			name = "Show Claimable Dynamite",
+			description = "Show a message when you can collect Dynamite from Thirus."
```
Comment (devLotto) [inline runelite-client/src/main/java/net/runelite/client/plugins/dailytaskindicators/DailyTasksConfig.java]: One tab for the indentation here, not two

**Comment by langhillie** [comment]
Per #8308, a relog might not be needed. I will confirm tonight and commit the change if necessary.

**Comment by langhillie** [comment]
Fixed merge conflict with #8375

**Pull request merged**